apps/experiments/models.py

```diff
@@ -714,7 +728,7 @@ def create_new_version(self, version_description: str | None = None, make_defaul
         new_version.version_number = version_number
 
         self._copy_attr_to_new_version("source_material", new_version)
-        self._copy_attr_to_new_version("consent_form", new_version)
+        self._copy_attr_to_new_version("consent_form", new_version, update_fields={"is_default": False})
```

Comment:
Since this is only used in one place what about just overriding 'create_new_version' in the consent form model to set it to False?

Reply:
Could do!

Reply:
https://github.com/dimagi/open-chat-studio/pull/979/commits/67d90cb8860031cbc62c7a1c66f9d7ea60e681bc